README.md

```diff
@@ -1,24 +1,218 @@
 
-* System dependencies
+After you're finished please remove all the comments and instructions!
+-->
 
-* Configuration
+<div align="center">
+  <br/>
+  <!-- <img src="" alt="logo" width="140"  height="auto" /> -->
+  <h1><b>setup RoR + React</b></h1>
 
-* Database creation
+</div>
 
-* Database initialization
+<!-- TABLE OF CONTENTS -->
 
-* How to run the test suite
+# 📗 Table of Contents
 
-* Services (job queues, cache servers, search engines, etc.)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Deployment instructions
+<!-- PROJECT DESCRIPTION -->
 
-* ...
+# 📖 [setup RoR + React] <a name="about-project"></a>
+
+> **[setup RoR + React]**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This is a projuct created to understand the pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.
+
+<details>
+  <summary><a href="https://github.com/betses/-setup-RoR-React-">Client / Frontend</a></summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Reactjs</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary><a href="https://github.com/betses/setup-ror-backend" >Server | Backend</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[View Events]**'
+  view different events.
+
+- **[Sign Up and Login ]**'
+  sign up and login to your account.
+
+- **[Reserve events]**'
+  reserve events that interest you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo API Endpoint<a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+git
+node
+npm
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/betses/-setup-RoR-React-.git
+  cd Final-capstone--frontend
+  Open in your favorite editor
+```
```

Comment:
- [ ] This is a`setup-ror-backend` repo and not the front-end repo. Kindly change the url link 👍🏻

Reply:
Thanks, @betses I have edited the readme file.